### PR TITLE
Fix passing of stale token

### PR DESCRIPTION
**Pull request by aaclan-ebi**
Related change in the ingest-client: https://github.com/ebi-ait/ingest-client/commit/e61bd21f15056665cc27b4884bbe43506188635f
(Apologies, I should have created a PR first!)

ebi-ait/dcp-ingest-central#13
ebi-ait/dcp-ingest-central#396

### CODE REVIEW

File: broker/submissions/export_to_spreadsheet_service.py [hunk 1]
```diff
@@ -6,11 +6,12 @@
...
 
     def __init__(self, ingest_api: IngestApi = None):
-        self.ingestapi = IngestApi() if not ingest_api else ingest_api
-        self.data_collector = DataCollector(self.ingestapi)
+        self.ingest_api = IngestApi() if not ingest_api else ingest_api
+        self.data_collector = DataCollector(self.ingest_api)
         self.downloader = XlsDownloader()
 
     def export(self, submission_uuid: str):
+        self.ingest_api.unset_token()
```
Comment (ke4) [inline broker/submissions/export_to_spreadsheet_service.py:14]: Why are we removing the token?
We just have to get a valid token if it expired or close to expiration

**Review by ke4 — CHANGES_REQUESTED** [review]
I added a comment about token management. Please consider it. Thanks

File: broker/submissions/export_to_spreadsheet_service.py (hunk 1 above)
Comment (ke4) [reply broker/submissions/export_to_spreadsheet_service.py:14]: Ok, we discussed offline that this is a temp solution to able to go ahead with [#13](https://app.zenhub.com/workspaces/dcp-ingest-product-development-5f71ca62a3cb47326bdc1b5c/issues/ebi-ait/dcp-ingest-central/13) dev ticket, so I am going to approve this

**Review by ke4 — APPROVED** [review]
fine for a temp solution for now to be able to go ahead with [#13](https://app.zenhub.com/workspaces/dcp-ingest-product-development-5f71ca62a3cb47326bdc1b5c/issues/ebi-ait/dcp-ingest-central/13)

**Review by yusra-haider — APPROVED** [review]

**Pull request merged**